modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -0,0 +1,295 @@
+installation of the OpenStack overcloud or upgraded if the overcloud already exists.
+The following steps provide basic non-custom steps in enabling Octavia and
+apply to both either a clean install of the overcloud or an overcloud update.
+
+[NOTE]
+The following steps only capture the key pieces required during the
+deployment of OpenStack when dealing with Octavia. For more information visit
+the documentation of https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/[Installation of OpenStack].
+It is also important to note that registry methods vary.
+For more information visit the documentation on https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/configuring-a-container-image-source#registry-methods[Registry Methods].
+This example used the local registry method.
+
+If using the local registry, create a template to upload the images to the
+registry. Example shown below.
+
+----
+(undercloud) $ openstack overcloud container image prepare \
+-e /usr/share/openstack-tripleo-heat-templates/environments/services-docker/octavia.yaml \
+--namespace=registry.access.redhat.com/rhosp13 \
+--push-destination=<local-ip-from-undercloud.conf>:8787 \
+--prefix=openstack- \
+--tag-from-label {version}-{release} \
+--output-env-file=/home/stack/templates/overcloud_images.yaml \
+--output-images-file /home/stack/local_registry_images.yaml
+----
+
+Verify that the created *_local_registry_images.yaml_* contains the Octavia images.
+
+.Octavia images in local registry file
+----
+...
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-api:13.0-43
+  push_destination: <local-ip-from-undercloud.conf>:8787
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-health-manager:13.0-45
+  push_destination: <local-ip-from-undercloud.conf>:8787
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-housekeeping:13.0-45
+  push_destination: <local-ip-from-undercloud.conf>:8787
+- imagename: registry.access.redhat.com/rhosp13/openstack-octavia-worker:13.0-44
+  push_destination: <local-ip-from-undercloud.conf>:8787
+----
+
+[NOTE]
+The versions of the Octavia containers will vary depending upon the specific
+{rh-openstack} release installed.
+
+
+The following step pulls the container images from registry.access.redhat.com
+to the undercloud node. This may take some time depending on the speed of the
+network and undercloud disk.
+
+----
+(undercloud) $ sudo openstack overcloud container image upload \
+  --config-file  /home/stack/local_registry_images.yaml \
+  --verbose
+----
+
+
+As an Octavia Load Balancer is used to access the OpenShift API, there is a
+need to increase their listeners default timeouts for the connections.
+The default timeout is 50 seconds. Increase the timeout to 20 minutes by
+passying the following file to the overcloud deploy command:
```

Comment:
"Since an Octavia Load Balancer is used to access the OpenShift API, you must increase their listeners' default timeouts for the connections. The default timeout is 50 seconds. Increase the timeout to 20 minutes by passing the following file to the Overcloud deploy command:"